src/SCREENS.ts

```diff
@@ -17,6 +17,7 @@ export default {
         WORKSPACES: 'Settings_Workspaces',
         SECURITY: 'Settings_Security',
         STATUS: 'Settings_Status',
+        WALLET: 'Settings_Wallet',
```

Comment:
Please use the screen constant where the screen name is actually defined [here](https://github.com/Expensify/App/blob/5cf1904d08ea09c70db40151a14406dd5c1d956c/src/libs/Navigation/AppNavigator/ModalStackNavigators.js#L143)